src/client/mapblock_mesh.cpp

```diff
@@ -729,12 +703,10 @@ MapBlockMesh::MapBlockMesh(Client *client, MeshMakeData *data, v3s16 camera_offs
 				tex.MagFilter = video::ETMAGF_NEAREST;
 			});
 
-			if (m_enable_shaders) {
+			{
 				material.MaterialType = m_shdrsrc->getShaderInfo(
 						p.layer.shader_id).material;
 				p.layer.applyMaterialOptionsWithShaders(material);
```

Comment:
Should we rename that function and remove the "withShaders" part?

Reply:
`applyMaterialOptions` is still in use, unfortunately.